DAS/das/mqtt_recorder_V2.py

```diff
@@ -52,22 +52,25 @@
     args = parser.parse_args()
 
     # Make logger object and initiate logging
-    main_recorder = logger.Record(
+    main_recorder = logger.Recorder(
         CSV_FILEPATH, topics=args.topics, broker_address=args.host, verbose=args.verbose
     )
 
-    # Nicely quits the recorder to ensure the data is saved
-    def graceful_exit(recorder):
-        recorder.stop()
-        sys.exit()
+    # Start the logger
+    main_recorder.start()
 
     # Logger can run forever or for a specific time
-    if args.time != float("Inf"):
-        time.sleep(args.time)
-        graceful_exit(main_recorder)
-    else:
-        while True:
-            try:
+    try:
+        if args.time != float("Inf"):
+            time.sleep(args.time)
+        else:
+            while True:
                 pass
-            except KeyboardInterrupt:
-                graceful_exit(main_recorder)
+
+    except (KeyboardInterrupt, Exception):
+        pass
```

Comment:
This means if an exception is thrown we won't ever get to see what happened. Maybe something like
```python
    except KeyboardInterrupt:
        pass
    except Exception as e:
        print(f"{type(e)}: {e}")
```
I know we're not likely to have anything go wrong in that try section but I'm just a little scared by catching all exceptions and ignoring them.

Reply:
100% agree will implement!